manage.py

```diff
@@ -65,5 +68,25 @@ def createAdmin(fname="robAdmin", lname="Smith", email="robadmin@mail.com", pass
     register_admin(fname, lname, email, password)
     print(fname+' created!')
 
+# CREATE ADMIN 
+@manager.command
+def createAdmin():
+    n1 = input('Enter ADMIN firstname :')
+    print (n1)
+    n2 = input('Enter ADMIN lastname :')
+    print (n2)
+    e1 = input('Enter ADMIN email :')
+    print (e1)
+    p1 = input('Enter ADMIN password :')
+    print (p1)
+
+    fname1 = User(first_name=n1)
```

Comment:
lines 83-84 creates 4 user objects and does nothing with it?

Reply:
I have fixed this, hope this is correct.

    User (first_name = n1)
    User (last_name = n2)
    User (email = e1)
    User (password = p1)

Will make changes to GitHub once I have finished the other corrections